doc/man3/OPENSSL_malloc.pod

```diff
@@ -155,15 +156,6 @@ the B<CRYPTO_MEM_CHECK_ON>.
 To disable tracking call CRYPTO_mem_ctrl() with a B<mode> argument of
 the B<CRYPTO_MEM_CHECK_OFF>.
 
-While checking memory, it can be useful to store additional context
-about what is being done.
-For example, identifying the field names when parsing a complicated
-data structure.
-OPENSSL_mem_debug_push() (which calls CRYPTO_mem_debug_push())
```

Comment:
I't ok to remove this chunk, but there should be a at least one sentence as replacement telling the user that those two functions don't do anything (anymore).

Reply:
added a sentence, let me know if it's okay.